src/common/redis_module/ray_redis_module.c

```diff
@@ -972,6 +1048,11 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx,
     return REDISMODULE_ERR;
   }
 
+  if (RedisModule_CreateCommand(ctx, "ray.disconnect", Disconnect_RedisCommand,
+                                "write", 0, 0, 0) == REDISMODULE_ERR) {
```

Comment:
hmm, should both this (and `ray.connect`) be `"write pubsub"` instead of just `"write"`?